mechanisms/bbp/K_Tst.mod

```diff
@@ -0,0 +1,55 @@
+: Comment    The transient component of the K current
+: Reference  Voltage-gated K+ channels in layer 5 neocortical pyramidal neurones from young rats:subtypes and gradients,Korngreen and Sakmann, J. Physiology, 2000
+: Comment:   shifted -10 mv to correct for junction potential
+: Comment    corrected rates using q10 = 2.3, target temperature 34, orginal 21
+
+NEURON {
+    SUFFIX K_Tst
+    USEION k READ ek WRITE ik
+    RANGE gK_Tstbar
+}
+
+UNITS {
+    (S)  = (siemens)
+    (mV) = (millivolt)
+    (mA) = (milliamp)
+}
+
+PARAMETER {
+    gK_Tstbar = 0.00001 (S/cm2)
+}
+
+STATE {
+    m
+    h
+}
+
+BREAKPOINT {
+    SOLVE states METHOD cnexp
+    ik = gK_Tstbar*(m^4)*h*(v-ek)
```

Comment:
m^4 will call exp, which might be slower than neccessary. I would not change this here, but consider making a patch to modcc instead.

Reply:
I think that might be more trouble than it's worth.

Reply:
That might also be true